src/Sql/dbo/Stored Procedures/CollectionCipher_ReadSharedByOrganizationId.sql

```diff
@@ -0,0 +1,16 @@
+CREATE PROCEDURE [dbo].[CollectionCipher_ReadSharedByOrganizationId]
+    @OrganizationId UNIQUEIDENTIFIER
+AS
+BEGIN
+    SET NOCOUNT ON
+
+    SELECT
+        CC.*
```

Comment:
It’s normally wise to only select the columns you need. Even if we need all the columns, we should be explicit about it so that when new columns are added, they don’t automatically get returned.

With that said, I’ve seen companies intentionally use `*` as part of a pattern, such as when a stored procedure is meant to be all-purpose. There are trade-offs to this, of course.